test/plugin/test_out_detect_exceptions.rb

```diff
@@ -98,6 +99,54 @@ def test_exception_detection
     assert_equal(make_logs(t, *messages), d.events)
   end
 
+  def test_ignore_exception_nested_in_json
+    cfg = 'languages python'
+    d = create_driver(cfg)
+    t = Time.now.to_i
+
+    # There is a nested exception within the body, we should ignore those!
+    lines = <<END
+{"timestamp": {"nanos": 998152494, "seconds": 1496420064}, "message": "Traceback (most recent call last):\\n  File \"<stdin>\", line 1, in <module>\\n  File \"/Library/Python/2.7/site-packages/requests-2.17.3-py2.7.egg/requests/api.py\", line 72, in get\\n    return request('get', url, params=params, **kwargs)\\n  File \"/Library/Python/2.7/site-packages/requests-2.17.3-py2.7.egg/requests/api.py\", line 58, in request\\n    return session.request(method=method, url=url, **kwargs)\\n  File \"/Library/Python/2.7/site-packages/requests-2.17.3-py2.7.egg/requests/sessions.py\", line 513, in request\\n    resp = self.send(prep, **send_kwargs)\\n  File \"/Library/Python/2.7/site-packages/requests-2.17.3-py2.7.egg/requests/sessions.py\", line 617, in send\\n    adapter = self.get_adapter(url=request.url)\\n  File \"/Library/Python/2.7/site-packages/requests-2.17.3-py2.7.egg/requests/sessions.py\", line 708, in get_adapter\\n    raise InvalidSchema(\"No connection adapters were found for '%s'\" % url)\\nrequests.exceptions.InvalidSchema: No connection adapters were found for 'htttp://www.google.com'", "thread": 139658267147048, "severity": "ERROR"}
```

Comment:
Since you repeat these verbatim, probably makes sense to make these an array of entries and iterate, rather than manually list each one. This applies to all of the tests below.